src/components/Web3ReactManager/index.tsx

```diff
@@ -19,24 +19,24 @@ const Message = styled.h2`
 `
 
 export default function Web3ReactManager({ children }: { children: JSX.Element }) {
-  const { active } = useWeb3React()
-  const { active: networkActive, error: networkError, activate: activateNetwork } = useWeb3React(NetworkContextName)
+  const { isActive } = useWeb3React()
+  const { isActive: networkActive, error: networkError, activate: activateNetwork } = useWeb3React(NetworkContextName)
 
   // try to eagerly connect to an injected provider, if it exists and has granted access already
   const triedEager = useEagerConnect()
```

Comment:
ah, so this is another piece of code that's unnecessary - in v8 eager connections can be automatic - for example if we pass the `connectEagerly` flag to the metamask provider (https://github.com/NoahZinsmeister/web3-react/blob/3e0a0785f27a1a0413a2f81d2b0669c017bd4bf0/packages/metamask/src/index.ts#L36) this logic will happen automatically on page load. there's some nuance if you're doing SSR, but we're not, so no worries there.

you should be able to pass this `connectEager` flag to any connector which supports the pattern

Reply:
i think it's just metamask for eager, but lmk if otherwise